controllers/v1/observationSubmissionsController.js

```diff
@@ -772,7 +783,7 @@ module.exports = class ObservationSubmissions extends Abstract {
   }
 
   /**
-  * @api {get} /assessment/api/v1/observationSubmissions/rate/:entityExternalId?solutionId=:solutionExternalId&createdBy=:keycloakUserId&submissionNumber=:submissionInstanceNumber Rate a Single Entity of Observation
+  * @api {get} /assessment/api/v1/observationSubmissions/rate/:entitycode?solutionId=:solutionExternalId&createdBy=:keycloakUserId&submissionNumber=:submissionInstanceNumber Rate a Single Entity of Observation
```

Comment:
@VISHNUDAS-tunerlabs please revert it to entityExternalId. don't change the request key, app won't change the req values